facebook/react-native
# update android ic_launcher

Update the android app icon with the new version.

## Test Plan
1. Create a new project using react-native init MyProject
2. Observe the app icon on android

## Release Notes
[ANDROID] [MINOR] [ic_launcher.png] - Update the Android app icon with the new version
